content/guides/ecommerce-connection/step3-reconcile-PIM-data-with-eCommerce-data.md

```diff
@@ -306,7 +318,12 @@ It is, therefore, possible to create these typical product associations:
 But also "custom" product association types for other needs.
 
 **What do you need to implement?**
-You should need to create a specific configuration in your connector to identify which PIM association type is a “Cross-sell”, “Up-sell”, “Substitution” or “Pack” association type to be able to map these associations in your eCommerce solution.
+
+Well, again, it depends on your customers needs.
+
+If your customers don't manage associations directly in the eCommerce solution and want to manage it manually in the PIM, you can create a specific configuration in your connector to identify which PIM association type is a “Cross-sell”, “Up-sell”, “Substitution” or “Pack” association type to be able to map these associations in your eCommerce solution.
```

Comment:
```suggestion
If your customers don't manage associations directly in the eCommerce solution and want to manage them manually in the PIM, you can create a specific configuration in your connector. It will allow you to identify which PIM association type is a “Cross-sell”, “Up-sell”, “Substitution” or “Pack” association type to be able to map these associations in your eCommerce solution.
```